### DISCUSSION TITLE
Performance

### DISCUSSION THREAD

**Discussion by contextlimit** [stashapp/stash#4521] [General]
It takes 5-10 minutes to load the dashboard with a filter. It takes 5-10 minutes for playback to start. Trying to load more than ~5 tabs doesn't work. I can search for and play the exact same video in Plex and its fine. Do we have a performance guide anywhere?

**Comment by WithoutPants** [comment]
Please check the network tab of the browser dev tools window (F12 usually), and look for requests that are taking a long time.
